src/scilpy/cli/scil_tracking_local_dev.py

```diff
@@ -156,9 +156,20 @@ def _build_arg_parser():
                         'Region-Adaptive Propagation tractography will start within '
                         'this mask.')
     track_g.add_argument('--rap_method', default='None',
-                        choices=['None', 'continue'],
-                        help="Region-Adaptive Propagation tractography method "
+                        choices=['None', 'continue', 'quack'],
+                        help="Region-Adaptive Propagation tractography method." \
```

Comment:
The backslashes are not required.
But please add some new lines in the text (\n)